docs/wiki/Contribution guide - Contribution flow.md

```diff
@@ -74,7 +79,7 @@ Finally, once your contribution is implemented and validated, open a PR for the
   - `[Fixes]`: For bug fixes when not specifically related to any of the above categories.
 - Provide a meaningful PR description.
 - Follow instructions you find in the PR template.
-- Set your Github handle as assignee.
+- Set your GitHub handle as assignee.
```

Comment:
Remove line since many especially new contributors would not have proper rights to self-assign + we have the autoassign pipeline (which I'd not mention here)
```suggestion
```